pyvista/core/dataset.py

```diff
@@ -1900,32 +1900,34 @@ def find_closest_cell(self, point: Union[int, np.ndarray]) -> Union[int, np.ndar
 
         Examples
         --------
-        Find nearest cell to a point on a sphere, centered on the
-        origin.
+        Find nearest cell on a sphere centered on the
+        origin to the point [0.1, 0.2, 0.3].
 
         >>> import pyvista
         >>> mesh = pyvista.Sphere()
-        >>> index = mesh.find_closest_cell([0, 0, 0.5])
+        >>> point = [0.1, 0.2, 0.3]
+        >>> index = mesh.find_closest_cell(point)
         >>> index
-        30
+        591
+
+        Make sure that this cell indeed is the closest to
+        (0.1, 0.2, 0.3).
```

Comment:
Minor nitpick: I'd use either list or tuple syntax in these two sentences. And maybe add double backticks for code formatting, not sure about that one.

Reply:
I think I slightly prefer double backticks since it is used directly in the code.